### ISSUE TITLE
Rename f^* to f^{-1}

### ISSUE THREAD

**Issue by felixwellen** [felixwellen/synthetic-zariski#26]
What we call $f^*$ now is actually a topological pullback and we should use the traditional notation $f^{-1}$. (Comment of @mnieper after my talk today).

**Comment by felixwellen** [comment]
We can also try to find out if we have a synthetic $f^*$

**Comment by MatthiasHu** [comment]
Where is the notation $f^*$ used for example?

**Comment by iblech** [comment]
Hm, I haven't watched the talk yet, but I believe that for a map $f : X \to Y$ of schemes and a bundle $E = (E_y)_{y:Y}$ of modules on E, the bundle $(E_{f(x)})_{x:X}$ on $X$ should definitely be called $f^*E$ and not $f^{-1}E$, as it is already a module over the structure ring of $X$ (namely the constant ring bundle $(R)_{x:X}$).

In classical algebraic geometry, $f^{-1}E$ is a sheaf of modules over $f^{-1}\mathcal{O}_Y$, and only after tensoring $\cdot \otimes_{f^{-1}\mathcal{O}_Y} \mathcal{O}_X$ do we obtain $f^*E$. But we don't need this additional tensoring. In some sense it's already built in.

**Comment by felixwellen** [comment]
Interesting. This is the definition of $f^*$ I was using. btw the talk does not really have anything on $f^*$ beyond the definition. I just warned that $f^*$ and $f_*$ behave not like the classical operations. In particular $f^*$, e.g. as a functor between wqc-bundles is exact - as $f^{-1}$ would be, but not the classic $f^*$.

**Comment by mnieper** [comment]
> Hm, I haven't watched the talk yet, but I believe that for a map f:X→Y of schemes and a bundle E=(Ey)y:Y of modules on E, the bundle (Ef(x))x:X on X should definitely be called f∗E and not f−1E, as it is already a module over the structure ring of X (namely the constant ring bundle (R)x:X).
> 
> In classical algebraic geometry, f−1E is a sheaf of modules over f−1OY, and only after tensoring ⋅⊗f−1OYOX do we obtain f∗E. But we don't need this additional tensoring. In some sense it's already built in.

The classical counterpart of bundles $E = (E_y)_{y : Y}$ is the _espace étale_ of a sheaf. The topological pullback of the espace étale $E$ along a continuous map $f\colon X \to Y$ is fibrewise given by $(E_{f(x)})_{x:X}$. Moreover, the constant ring bundle over $X$ is just the topological pullback of the constant ring bundle of the bundle over $Y$.

In other words, the synthetic pullback of bundles behaves as the topological pullback of (the espace étale of) sheaves. The latter is denoted by $f^{-1}$ in the context of algebraic geometry and is exact. As we discussed earlier, we should not fall into the trap of viewing quasicoherent sheaves as the classical counterpart of our synthetic bundles.

**Comment by mnieper** [comment]
Verbatim because the LaTeX formatter is broken:
```
> Hm, I haven't watched the talk yet, but I believe that for a map f:X→Y of schemes and a bundle E=(Ey)y:Y of modules on E, the bundle (Ef(x))x:X on X should definitely be called f∗E and not f−1E, as it is already a module over the structure ring of X (namely the constant ring bundle (R)x:X).
> 
> In classical algebraic geometry, f−1E is a sheaf of modules over f−1OY, and only after tensoring ⋅⊗f−1OYOX do we obtain f∗E. But we don't need this additional tensoring. In some sense it's already built in.

The classical counterpart of bundles $E = (E_y)_{y : Y}$ is the _espace étale_ of a sheaf. The topological pullback of the espace étale $E$ along a continuous map $f\colon X \to Y$ is fibrewise given by $(E_{f(x)})_{x:X}$. Moreover, the constant ring bundle over $X$ is just the topological pullback of the constant ring bundle of the bundle over $Y$.

In other words, the synthetic pullback of bundles behaves as the topological pullback of (the espace étale of) sheaves. The latter is denoted by $f^{-1}$ in the context of algebraic geometry and is exact. As we discussed earlier, we should not fall into the trap of viewing quasicoherent sheaves as the classical counterpart of our synthetic bundles. 
```

**Comment by DavidMichaelRoberts** [comment]
You can always write $\ast blah \ast$ (`$\ast blah \ast$`) instead of $*blah*$ (`$*blah*$`) in order to not get the Markdown parser confused...

**Comment by DavidMichaelRoberts** [comment]
BTW, where are the videos? I presume it is meant this years? Not on the same YT channel that I can see...
